test-requirements.in

```diff
@@ -8,6 +8,8 @@ trustme               # for the ssl + DTLS tests
 pylint                # for pylint finding all symbols tests
 jedi; implementation_name == "cpython"                  # for jedi code completion tests
 cryptography>=41.0.0  # cryptography<41 segfaults on pypy3.10
+# Temp workaround until parso makes a new release
+git+https://github.com/davidhalter/parso.git; python_version >= "3.14"
```

Comment:
Hopefully in the future (after https://github.com/davidhalter/parso/issues/232) this will not be necessary ever again